src/nbl/video/vulkan/profiles/NablaCore.json

```diff
@@ -1,2017 +1,2216 @@
-                    "shaderStorageTexelBufferArrayDynamicIndexing": true,
-                    "shaderSampledImageArrayNonUniformIndexing": true,
-                    "shaderStorageImageArrayNonUniformIndexing": true,
-                    "shaderUniformTexelBufferArrayNonUniformIndexing": true,
-                    "shaderStorageTexelBufferArrayNonUniformIndexing": true,
-                    "descriptorBindingSampledImageUpdateAfterBind": true,
-                    "descriptorBindingStorageImageUpdateAfterBind": true,
-                    "descriptorBindingStorageBufferUpdateAfterBind": true,
-                    "descriptorBindingUniformTexelBufferUpdateAfterBind": true,
-                    "descriptorBindingStorageTexelBufferUpdateAfterBind": true,
-                    "descriptorBindingUpdateUnusedWhilePending": true,
-                    "descriptorBindingPartiallyBound": true,
-                    "descriptorBindingVariableDescriptorCount": true,
-                    "runtimeDescriptorArray": true,
-                    "scalarBlockLayout": true,
-                    "imagelessFramebuffer": true,
-                    "uniformBufferStandardLayout": true,
-                    "shaderSubgroupExtendedTypes": true,
-                    "separateDepthStencilLayouts": true,
-                    "hostQueryReset": true,
-                    "timelineSemaphore": true,
-                    "bufferDeviceAddress": true,
-                    "subgroupBroadcastDynamicId": true
-                }
-            },
-            "properties": {
-                "VkPhysicalDeviceProperties": {
-                    "sparseProperties": {
-                        "residencyAlignedMipSize": true,
-                        "residencyStandard2DBlockShape": true,
-                        "residencyStandard3DBlockShape": true,
-                        "residencyNonResidentStrict": true
-                    },
-                    "limits": {
-                        "bufferImageGranularity": 65536,
-                        "discreteQueuePriorities": 2,
-                        "framebufferColorSampleCounts": [
-                            "VK_SAMPLE_COUNT_1_BIT",
-                            "VK_SAMPLE_COUNT_2_BIT",
-                            "VK_SAMPLE_COUNT_4_BIT"
-                        ],
-                        "framebufferDepthSampleCounts": [
-                            "VK_SAMPLE_COUNT_1_BIT",
-                            "VK_SAMPLE_COUNT_2_BIT",
-                            "VK_SAMPLE_COUNT_4_BIT"
-                        ],
-                        "framebufferNoAttachmentsSampleCounts": [
-                            "VK_SAMPLE_COUNT_1_BIT",
-                            "VK_SAMPLE_COUNT_2_BIT",
-                            "VK_SAMPLE_COUNT_4_BIT"
-                        ],
-                        "framebufferStencilSampleCounts": [
-                            "VK_SAMPLE_COUNT_1_BIT",
-                            "VK_SAMPLE_COUNT_2_BIT",
```

Comment:
see in Nabla if we have any code that requires/assumes one can do at least 4 samples (2 included)

Reply:
also all profiles supports 1-8 samples (only apple is 1-4)

Reply:
actually looks like some don't support 2

Reply:
> no examples use sample count higher than 1

I meant some nabla validation code etc. on the renderpasses and framebuffers, not examples.